tinyxml2.cpp

```diff
@@ -2304,6 +2282,17 @@ void XMLDocument::Parse()
     ParseDeep(p, 0, &_parseCurLineNum );
 }
 
+template<class NodeType, int PoolElementSize>

```

Comment:
I think so. What the practical support is for this sort of template isn't clear to me. But moving it up the cpp file will theoretically help.